client/my-sites/plan-features/index.jsx

```diff
@@ -138,10 +138,7 @@ class PlanFeatures extends Component {
 				isPlaceholder
 			} = properties;
 			let { rawPrice, discountPrice } = properties;
```

Comment:
Now that we're not reassigning `rawPrice` and `discountPrice`, we can change that `let` to `const`.